cypress/e2e/title-level-request/user-can-move-title-level-request-from-one-item-to-another.cy.js

```diff
@@ -0,0 +1,166 @@
+            materialType: { id: testData.materialTypeId },
+          },
+        ];
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: testData.instanceTypeId,
+            title: instanceData.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: testData.holdingTypeId,
+              permanentLocationId: testData.defaultLocation.id,
+            },
+          ],
+          items: instanceData.itemsData,
+        }).then((specialInstanceIds) => {
+          instanceData.instanceId = specialInstanceIds.instanceId;
+          instanceData.holdingId = specialInstanceIds.holdingIds[0].id;
+          instanceData.itemIds = specialInstanceIds.holdingIds[0].itemIds;
+        });
+        cy.wrap(instanceData.itemsData).as('itemsToMove');
+      });
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+
+    cy.createTempUser([Permissions.requestsAll.gui], patronGroup.name)
+      .then((userProperties) => {
+        userData = userProperties;
+        UserEdit.addServicePointViaApi(
+          testData.userServicePoint.id,
+          userData.userId,
+          testData.userServicePoint.id,
+        );
+      })
+      .then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+        Requests.createNewRequestViaApi({
+          fulfillmentPreference: FULFILMENT_PREFERENCES.HOLD_SHELF,
+          instanceId: instanceData.instanceId,
+          pickupServicePointId: testData.userServicePoint.id,
+          requestDate: new Date(),
+          requestLevel: REQUEST_LEVELS.TITLE,
+          requestType: REQUEST_TYPES.PAGE,
+          requesterId: userData.userId,
+        }).then((request) => {
+          testData.requestId = request.body.id;
+          testData.currentItem = request.body.item.barcode;
+        });
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    cy.loginAsAdmin({
+      path: SettingsMenu.circulationTitleLevelRequestsPath,
+      waiter: TitleLevelRequests.waitLoading,
+    });
+    Requests.deleteRequestViaApi(testData.requestId);
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(testData.currentItem);
+    cy.deleteLoanType(testData.loanTypeId);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id,
+    );
+    TitleLevelRequests.changeTitleLevelRequestsStatus('forbid');
```

Comment:
Can this be rewritten to use API call?

Reply:
till now,I didn't find this API to change title level request status if you have tlrEdit permission

Reply:
Let's skip this improvement